apps/homescreen/js/app.js

```diff
@@ -829,22 +829,16 @@
       case 'drag-end':
         if ((!this.draggingRemovable && !this.draggingEditable) ||
             e.detail.clientY <= window.innerHeight - DELETE_DISTANCE) {
-          // If the drop target is null, check to see if we're
-          // dropping over the icon itself, and if we aren't, we must be
-          // dropping over the start or end of the container.
-          if (!e.detail.dropTarget) {
-            var rect = this.icons.getChildOffsetRect(e.detail.target);
```

Comment:
Homescreen doesn't use it, but it's a generally useful container feature that I'd like to keep - I expect we'll end up using it when implementing more advanced features. I may be wrong about that, but let's audit functions at a separate time.